libraries/ESP8266WiFi/src/BearSSLHelpers.cpp

```diff
@@ -20,6 +20,7 @@
   Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 */
 
+#include "BearSSLHelpers.h"
```

Comment:
Why the move of this include before the others?

Reply:
I use this idiom as good practice to ensure that headers are self contained - I don't want other people having to find out the hard way that some header dependencies must be manually resolved each time.